rita/src/exit.rs

```diff
@@ -166,7 +173,14 @@ fn main() {
     // On Linux static builds we need to probe ssl certs path to be able to
     // do TLS stuff.
     openssl_probe::init_ssl_cert_env_vars();
-    env_logger::init();
+
+    // Log at info level by default
+    match env::var("RUST_LOG") {
+        Ok(_) => env_logger::init(),
+        Err(_) => env_logger::Builder::new()
```

Comment:
no can do, start on ERR even warn fills router storage in minutes.

Reply:
I suppose we can change the setting on the routers by updating the startup script with the package, thoughts?

Reply:
@jkilpatr err is the default